supervisord/datadog_checks/supervisord/config_models/defaults.py

```diff
@@ -1,7 +1,3 @@
-# (C) Datadog, Inc. 2021-present
```

Comment:
How did this get deleted?

Reply:
I looked at the CI/CD jobs that weren't success, some were talking about `ddev validate config/models` to fix.
I then ran these commands, which modified the files... I can restore these headers manually if you want ;)

Reply:
We need them back in order to fix another validation :)

So yes, if you could add them back, that would be great!

Reply:
Restored :)